packages/blocks/src/surface-block/surface-model.ts

```diff
@@ -136,6 +136,21 @@ const migration = {
       const wrapper = new Boxed(yMap);
       data.elements = wrapper;
     }
+
+    const childrenMap = data.elements.getValue() as Y.Map<Y.Map<unknown>>;
```

Comment:
I think we should avoid direct operating on YTypes.

Reply:
> I think we should avoid direct operating on YTypes.

This happens during migration. Not sure if there's better way. cc @Saul-Mirone